lib/MusicBrainz/Server/Controller/Admin.pm

```diff
@@ -35,22 +35,22 @@ sub edit_user : Path('/admin/user/edit') Args(1) RequireAuth HiddenOnSlaves
     my $form2 = $c->form(
         form => 'User::EditProfile',
         item => {
-            email            => $user->email,
+            username          => $user->name,
+            email             => $user->email,
             skip_verification => 0,
-            website            => $user->website,
-            biography        => $user->biography
+            website           => $user->website,
+            biography         => $user->biography
         },
     );
 
     if ($c->form_posted) {
-        if ($form->submitted_and_valid($c->req->params )) {
+        if ($form->submitted_and_valid($c->req->params) 
+            && $form2->submitted_and_valid($c->req->params )) {
```

Comment:
As far as I can tell, both forms *should* always be submitted at the same time. There's only one form element, with one submit button, pressing Enter anywhere submits both forms from what I've tested, and the separation into two forms seems to exist only in the FormHandler part of this. But I'm not super experienced with this, so I might be missing something. Am I?